docs/manual/java/gettingstarted/JavaPrereqs.md

```diff
@@ -46,23 +51,22 @@ To check the Maven version from a command line, enter:
 
 `mvn -version` 
 
-
-
 To install Maven, see the official [Maven installation page](https://maven.apache.org/install.html).
 
 ## sbt
 
-[sbt](http://www.scala-sbt.org) is a build tool for Java and Scala. Lagom requires sbt 0.13.5 or higher. To create new projects using Lagom-supplied templates, use sbt 0.13.13 or higher, which contains the sbt new command.
+[sbt](http://www.scala-sbt.org) is a build tool for Java and Scala. Lagom recommends using sbt 1.0 or higher.
 
-In a console, check your version using the `sbt sbt-version`command:
+In a console, check your version using the `sbt sbtVersion`command:
```

Comment:
dash syntax was causing issues on my machine when testing `sbt sbt-version` (not sure if that's phased out TBH).

Reply:
Dash syntax (or kebab syntax) was phased out in sbt 0.13, and dropped in sbt 1.